core/iwasm/interpreter/wasm_mini_loader.c

```diff
@@ -4623,6 +4632,7 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
                 }
 
                 func->has_op_func_call = true;
+                if (opcode == WASM_OP_RETURN_CALL_INDIRECT) { goto handle_op_return; }
```

Comment:
Same with the previous comment. Need to set a flag to avoid emitting operand when build as fast-interpreter

And same in `wasm_loader.c`

Reply:
Postpone this part to fast-interpreter implementation.